docs/user/04-00-istio-custom-resource.md

```diff
@@ -48,6 +48,7 @@ This table lists all the possible parameters of Istio CR together with their des
 | **config.authorizers**                                      | \[\]authorizer | Specifies the list of external authorizers configured in the Istio service mesh config.                                                                                                                                                                                                                                                       |
 | **config.numTrustedProxies**                                | integer        | Specifies the number of trusted proxies deployed in front of the Istio gateway proxy.                                                                                                                                                                                                                                                         |
 | **config.gatewayExternalTrafficPolicy**                     | string         | Defines the external traffic policy for Istio Ingress Gateway Service. Valid configurations are `Local` or `Cluster`. The external traffic policy set to `Local` preserves the client IP in the request but also introduces the risk of unbalanced traffic distribution.                                                                      |
+| **config.telemetry.metrics.prometheusMerge**                | bool           | Enables the prometheusMerge feature from Istio, which will merge application's and istio's metrics and expose them together at `:15020/stats/prometheus` for scraping using plain HTTP                                                                                                                                                        |
```

Comment:
```suggestion
| **config.telemetry.metrics.prometheusMerge**                | bool           | Enables the prometheusMerge feature from Istio, which merges the application's and Istio's metrics and exposes them together at `:15020/stats/prometheus` for scraping using plain HTTP.                                                                                                                                                        |
```

Not sure if this is the correct place to adjust this text - is this table manually maintained or auto-generated?